templates/zerver/login.html

```diff
@@ -141,6 +141,16 @@ <h1 class="get-started">{{ _("Log in to Zulip") }}</h1>
                             </div>
                             {% endif %}
 
+                            {% if saml_auth_enabled %}
+                            {% for idp in saml_get_idps %}
+                            <div class="login-saml">
+                                <form class="form-inline saml-wrapper" action="{{ url('login-saml', kwargs={"backend":"saml", "idp":idp}) }}" method="get">
+                                    <button class="login-social-button login-saml-button saml">{{ _('Log in with %(idp)s', idp=idp) }}</button>
```

Comment:
I think it might be a good idea from an i18n perspective to change the strings for GitHub and Google to use `Log in with %(identity_provider)` as the translation string, and thus have this as those use the same string.

(We generally prefer to avoid abbreviations in variable names in Zulip, hence the longer name in my example)

Reply:
I just merged b91de0e28328f371c74b506d956f6427b960a554 to master, which should eliminate the need for this PR to add any new translated strings.